Name 'nn.TransformerEncoderLayer' is not defined

## 🐛 Bug

benchmarks/transformer.py:43: error: Name 'nn.TransformerEncoderLayer' is not defined

## Command

## To Reproduce

Steps to reproduce the behavior:

1. conda install -c conda-forge pre-commit
2. pre-commit run --all-files

## Environment

[pip3] numpy==1.18.5
[pip3] torch==1.5.1
[pip3] torchvision==0.6.0a0+35d732a
[conda] blas                      1.0                         mkl
[conda] mkl                       2019.4                      243
[conda] mkl-service               2.3.0            py38h516909a_0    conda-forge
[conda] mkl_fft                   1.1.0            py38hc1659b7_1    conda-forge
[conda] mkl_random                1.1.0            py38h962f231_0
[conda] pytorch                   1.5.1           py3.8_cuda10.1.243_cudnn7.6.3_0    pytorch
[conda] torchvision               0.6.1                py38_cu101    pytorch

--- a/stubs/torch/nn/modules/__init__.pyi
+++ b/stubs/torch/nn/modules/__init__.pyi
@@ -44,6 +44,7 @@ from .pooling import AdaptiveAvgPool1d as AdaptiveAvgPool1d, AdaptiveAvgPool2d a
 from .rnn import GRU as GRU, GRUCell as GRUCell, LSTM as LSTM, LSTMCell as LSTMCell, RNN as RNN, RNNBase as RNNBase, \
     RNNCell as RNNCell, RNNCellBase as RNNCellBase
 from .sparse import Embedding as Embedding, EmbeddingBag as EmbeddingBag
+from .transformer import TransformerEncoderLayer as TransformerEncoderLayer
 from .upsampling import Upsample as Upsample, UpsamplingBilinear2d as UpsamplingBilinear2d, \
     UpsamplingNearest2d as UpsamplingNearest2d
 

